AWSPinpoint/AWSPinpointEndpointProfile.h

```diff
@@ -276,7 +276,35 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, strong) NSString *_Nullable userId;
 
+/**
+ Returns an Dictionary of all user attributes contained within this AWSPinpointEndpointProfileUser
+ @returns an Dictionary of all user attributes, where the user attribute keys are the keys and the user attribute values are the values
+ */
+- (NSDictionary *)allUserAttributes;
+
+/**
+Adds an user attribute to this AWSPinpointEndpointProfileUser with the specified key. Only 40 attributes
+are allowed to be added to an AWSPinpointEndpointProfileUser. If 40 user attributes already exist on this AWSPinpointEndpointProfileUser, the call is ignored.
```

Comment:
How did we decide that we should ignore the call?

Reply:
Also I am trying to keep parallel with the [existing methods](https://github.com/aws-amplify/aws-sdk-ios/blob/main/AWSPinpoint/AWSPinpointEndpointProfile.h#L104-L111). I don't find anywhere saying `Only 40 attributes
are allowed to be added to an AWSPinpointEndpointProfile`